packages/compose/src/index.js

```diff
@@ -1,20 +1,8 @@
-/**
- * External dependencies
- */
-import { flowRight } from 'lodash';
-
 // Utils
 export { default as createHigherOrderComponent } from './utils/create-higher-order-component';
 
-/**
- * Composes multiple higher-order components into a single higher-order component. Performs right-to-left function
- * composition, where each successive invocation is supplied the return value of the previous.
- *
- * @param {...Function} hocs The HOC functions to invoke.
- *
- * @return {Function} Returns the new composite function.
- */
-export { flowRight as compose };
+// Compose helper (aliased flowRight from Lodash)
+export { default as compose } from './higher-order/compose';
```

Comment:
Question: Was there a reason this couldn't be the following? Would it hinder tree shaking?

```suggestion
export { flowRight as compose } from 'lodash';
```

I'm not requesting this be changed, it seems fine to me as-is.

Reply:
Yes, that would also work. I tried to make the `index.js` file to contain only reexports and nothing else, and move all implementation (It can be argued that the use `lodash` is an implementation detail) and documentation (I also need to put the jsdoc comment somewhere) to other modules.

Reply:
I'd love to learn more as well. I'm fine leaving it as is but at the same time I can't figure out why it would make a difference.